.github/workflows/test.yaml

```diff
@@ -73,3 +73,10 @@ jobs:
         with:
           base: https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test1.yaml
           revision: https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test3.yaml
+      - name: Test changelog action output
+        run: |
+          output="${{steps.test_changelog.outputs.changelog}}"
+          if [[ -z "${output}" ]]; then
```

Comment:
Looks like it is always true, in case of empty output your script returns "No changelog changes".

Reply:
Does it look better?
https://github.com/axebeche/oasdiff-action/actions/runs/7446565018/job/20257084046#step:4:9
https://github.com/axebeche/oasdiff-action/actions/runs/7446592722/job/20257167451#step:4:2

Reply:
I prefer to conduct tests in the following manner. This approach ensures that even if someone modifies the script or empties the message, the test will still function. However, if there are changes made to the OpenAPI, the test will fail.
```
readonly expected_output="20 changes: 2 error, 4 warning, 14 info"
if [ "$output" != "$expected_output" ]; then
    echo "Expected output '$expected_output' but got '$output'" >&2
    exit 1
fi
```

Reply:
> However, if there are changes made to the OpenAPI, the test will fail.

Yea. This approach heavily tied to "20 changes: 2 error, 4 warning, 14 info" case.  But if  these
 ` base: https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test1.yaml
   revision: https://raw.githubusercontent.com/Tufin/oasdiff/main/data/openapi-test3.yaml`
will be changed with time?

Reply:
The test will fail and we will fix the test or the OAS :)

Reply:
Hi )
Why if you cut the output to one line
https://github.com/oasdiff/oasdiff-action/blob/e1c54f4fff1be7666d67b74b9567ebd325872d0b/breaking/entrypoint.sh#L46
you comparing that against 1 MB?
https://github.com/oasdiff/oasdiff-action/blob/e1c54f4fff1be7666d67b74b9567ebd325872d0b/breaking/entrypoint.sh#L53
And what is the $1000000?
https://github.com/oasdiff/oasdiff-action/blob/e1c54f4fff1be7666d67b74b9567ebd325872d0b/breaking/entrypoint.sh#L55

Reply:
It started with a big output, but decided to use only summary.
Github output limitation is 1MB.

Reply:
I mean may be $ is unnecessary?

Reply:
Right, thanks.
I'll remove this section for `breaking` and please remove the `$` in here.

Reply:
Removed. By this commit https://github.com/oasdiff/oasdiff-action/commit/83d2b23366839725b222f725a5b41fcf539c2de1 I see you prefer use awkward bin/test instead of built-in. What is the reason?

Reply:
I don't understand what do you mean.

Also, I'll be happy to merge this PR, can you fix the test?

Reply:
IMHO, it's good enough now.
https://github.com/axebeche/oasdiff-action/actions/runs/7474256170/job/20340034446#step:4:1